File: utils/VectorEnginePlugin.ref/pares/funciones/primTargetAssumedOpaque.js
function primTargetAssumedOpaque(argCount) {
    if (!isBooleanObject(stackValue(0))) {
        primitiveFailFor(PrimErrBadArgument); return false;
    }
    targetAssumedOpaque = booleanValueOf(stackValue(0));
    if (!failed()) pop(1);
    return !failed();
}

File: utils/VectorEnginePlugin.ref/pares/funciones/primTargetAssumedOpaque.c
/* VectorEnginePlugin>>#targetAssumedOpaque: */
EXPORT(sqInt)
primTargetAssumedOpaque(void)
{
	sqInt aBoolean;

	if (!(isBooleanObject(stackValue(0)))) {
		return primitiveFailFor(PrimErrBadArgument);
	}
	aBoolean = booleanValueOf(stackValue(0));
	targetAssumedOpaque = aBoolean;
	if (!(failed())) {
		pop(1);
	}
	return null;
}
